### PR TITLE
Implement SEO elements and Instruction on the theme (issue #113)

### LINKED ISSUE #113: Enhancement: Add SEO parameters to theme
## Expected Behavior

I suggest an improvement to implement SEO code to websites.
The implementation is following the description in the following blog:
[https://keithpblog.org/post/hugo-website-seo/](https://keithpblog.org/post/hugo-website-seo/)
I've implemented the changes on my own site and they are working fine and easy to set.
I'll be happy to submit a PR if relevant to the general public

## Current Behavior

The SEO parameter do not implemented today in theme and needed to be added manually.

## Possible Solution

If not implemented on theme level, one could follow the instructions above to set it manually.

## Steps to Reproduce or Template with Error (for bugs)

N/A

## Context

I was trying to push my site via the search engines and was looking for the right way to do so on Hugo
## Your Environment

**Hugo Version:**
0.35

**Browser:**
OSX/iOS Safari

**Desktop or Mobile:**
Both

**Link to your project:**
[https://elv8.co](https://elv8.co)

### PR DESCRIPTION
**Pull request by dudil**
- Add Hugo related SEO parameters to example config.toml
- Add relevant instructions to the README.md file

## Description

Following discussion on issue #113, I've added SEO elements and instructions on how to enable that on the website

## Motivation and Context

To allow easy implementation of SEO for theme users, Closes #113 
I've also modified the readme.md file to allow easy implementation using the Google/Bing consoles

## How Has This Been Tested?

**Hugo Version:**
0.35

**Browser(s):**
Safari

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the [code style] of this project.
- [X] My change requires a change to the [documentation].
- [X] I have updated the documentation, including `theme.toml`, accordingly.
- [X] I have read the [Contributing Document].

[Code Style]: https://github.com/jpescador/hugo-future-imperfect/blob/master/.github/CONTRIBUTING.md#Style-Guide
[Contributing Document]: https://github.com/jpescador/hugo-future-imperfect/blob/master/.github/CONTRIBUTING.md
[Documentation]: https://github.com/jpescador/hugo-future-imperfect/wiki

### CODE REVIEW

File: layouts/partials/seo_schema.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        "@type" : "BlogPosting",
+        "mainEntityOfPage": {
+             "@type": "WebPage",
+             "@id": "{{ .Site.BaseURL }}"
+        },
+        "articleSection" : "{{ .Section }}",
+        "name" : "{{ .Title }}",
+        "headline" : "{{ .Title }}",
+        "description" : "{{ if .Description }}{{ .Description }}{{ else }}{{if .IsPage}}{{ .Summary }}{{ end }}{{ end }}",
+        "inLanguage" : "en-US",
```
Comment (statnmap) [inline layouts/partials/seo_schema.html]: Can we use {{ .Lang }} here in case the main Language is not English ?

**Comment by statnmap** [comment]
Hi. Thanks for the proposition.
I think that the "seo_schema" only needs to appear in "partials/header.html" as this is the main header of the website.
The other headers are for articles or list of articles, which means that they are already under the main header of the website. If we include the "seo_schema" in these headers too, it will be repeated twice or more on pages, which may be worse for SEO.

_This is my quick view on it._

**Comment by dudil** [comment]
Ok.
Both comments make sense.
I’ll handle them and will make a new PR

On 11 Feb 2018, at 23:13, Sébastien Rochette <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi. Thanks for the proposition.
I think that the "seo_schema" only needs to appear in "partials/header.html" as this is the main header of the website.
The other headers are for articles or list of articles, which means that they are already under the main header of the website. If we include the "seo_schema" in these headers too, it will be repeated twice or more on pages, which may be worse for SEO.

This is my quick view on it.

**Comment by pacollins** [comment]
No need to make a new PR. just recommit to `feature/issue-113`. If you need to know how to reset, let me know.

**Comment by dudil** [comment]
@pacollins  - Done - I've recommit the changes.
@statnmap - Good comments, thank you for helping out.
Please let me know if there nay other issues.

**Pull request merged**